### PR TITLE
Fix some warnings in log.py.

### PR DESCRIPTION
**Pull request by ChiahungTai**
## Description

Fix some warnings in log.py.

## How Has This Been Tested?
- [x] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
Screen Shot 2022-01-05 at 3 53 45 AM

## Types of changes

- [x] Fix bugs
- [ ] Add new feature
- [ ] Update documentation

### CODE REVIEW

**Comment by ChiahungTai** [comment]
Screen Shot 2022-01-05 at 9 21 14 AM

Screen Shot 2022-01-05 at 9 21 30 AM

This is weird! I pass my black in my pycharm.
@you-n-g 
Is there any settings I missed for black?

**Comment by you-n-g** [comment]
Are you using the latest version of black?  @ChiahungTai 
image

**Comment by ChiahungTai** [comment]
> Are you using the latest version of black? @ChiahungTai image

Thanks for the diagram. Looks like I miss the argument -l 120.

File: qlib/log.py [hunk 1]
```diff
@@ -13,12 +13,12 @@
 
 
 class MetaLogger(type):
-    def __new__(cls, name, bases, dict):
+    def __new__(mcs, name, bases, dict_):
```
Comment (you-n-g) [inline qlib/log.py:16]: why do you rename `dict` to `dict_` ?
Can you give me any references for this naming convention?

Comment (ChiahungTai) [reply qlib/log.py:16]: To avoid overwriting the python keyword, the suggestion of use _ as postfix comes from the chapter 2-4 in the book Python Tricks.

Comment (you-n-g) [reply qlib/log.py:16]: I see now. Thanks to your advice
Will `attrs` be a better name than `dict_`?

Comment (ChiahungTai) [reply qlib/log.py:16]: LGTM for attrs. Will change to it.

**Comment by you-n-g** [comment]
@ChiahungTai  It looks great!
Thanks so much

**Pull request merged**